### DISCUSSION TITLE
Blocked instances: Wildcard characters supported?

### DISCUSSION THREAD

**Discussion by AlfredSK** [misskey-dev/misskey#9256] [Q&A] [answered]
Hello,
does Misskey support wildcard characters on the list of blocked instances (.../admin/instance-block)?

I want to add an entry like `*.exampledomain.tld` to block all subdomains of exampledomain.tld. Is that possible?

Thanks.

**Comment by ThatOneCalculator** [comment]
Wildcards are not implemented rn

**Comment by VewMet** [comment, marked as answer]
if you put abcd.xyz to blocked instances list, all subdomains are also blocked by default. For example, pqr.abcd.xyz will also be blocked automatically.

**Comment by AlfredSK** [comment]
Great! I was asking because of the activitypub-troll.cf madness.
Thanks.
